spring-data-relational/src/main/java/org/springframework/data/relational/core/sql/render/SqlRenderer.java

```diff
@@ -69,16 +71,6 @@ public static String toString(Select select) {
 		return create().render(select);
 	}
 
-	/**
```

Comment:
We should still have a way to render an Insert statement without dialect, falling back to sensible defaults.